docs/rules/quote-props.md

```diff
@@ -42,7 +42,7 @@ String option:
 
 * `"always"` (default) requires quotes around all object literal property names
 * `"as-needed"` disallows quotes around object literal property names that are not strictly required
-* `"consistent"` enforces a consistent quote style requires quotes around object literal property names
+* `"consistent"` enforces a consistent quote style
```

Comment:
Could this be expanded to something like:

> `"consistent"` enforces a consistent quote style; in a given object, either all of the properties should be quoted, or none of the properties should be quoted.

My concern is that "a consistent quote style" could be misinterpreted, given that all of the options enforce some level of "consistency" (on the level of an entire codebase rather than an individual object).

Reply:
Done.